securing_apps/topics/saml/java/tomcat-adapter/tomcat_adapter_per_war_config.adoc

```diff
@@ -16,6 +16,16 @@ This is a Tomcat specific config file and you must define a Keycloak specific Va
 Next you must create a `keycloak-saml.xml` adapter config file within the `WEB-INF` directory of your WAR.
 The format of this config file is described in the <<_saml-general-config,General Adapter Config>> section.
 
+In case a file location is required for `keycloak-saml.xml` (in cases of shared files and/or runtime configuration). It is possible to user a file location parameter  name "keycloak.config.file", as in the example bellow:
```

Comment:
The first sentence is incomplete. Please fix this by changing the period to a comma and change "It is possible to "it is possible..."
Also change "user" to "use" and "bellow" to "below"